index.js

```diff
@@ -12,16 +12,49 @@ mongoose
   .connect(MONGODB_URI, {
     useCreateIndex: true,
     useNewUrlParser: true,
-    useUnifiedTopology: true
+    useUnifiedTopology: true,
+    useFindAndModify: false
   })
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
     // Before adding any documents to the database, let's delete all previous entries
-    return self.connection.dropDatabase();
+    //return self.connection.dropDatabase();
   })
-  .then(() => {
-    // Run your code here, after you have insured that the connection was made
+   const recipeObject = {
+    "title": "Chocolate Chip Cookies",
+    "level": "Amateur Chef",
+    "ingredients": [
+      "1/2 cup light brown sugar",
+      "1 large egg",
+      "2 tablespoons milk",
+      "1 1/4 teaspoons vanilla extract",
+      "2 cups semisweet chocolate chips"
+    ],
+    "cuisine": "French",
+    "dishType": "dessert",
+    "image": "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fimages.media-allrecipes.com%2Fuserphotos%2F4398987.jpg&w=596&h=399.32000000000005&c=sc&poi=face&q=85",
+    "duration": 30,
+    "creator": "Chef Jennifer"
+  };
+  Recipe.create(recipeObject)
```

Comment:
Perhaps apply some empty lines between the method blocks to give the code some air.